Provide consistent interface for Synthetic Agents API

danielfurman opened the pull request:
Issue: KNTK-378

Commit:
Provide consistent interface for Synthetic Agents API

Issue: KNTK-378

mmac-m3a reviewed (approved):
LGTM

I'm not sure how changes in cloud export examples (and related API code) are relevant to "synthetic agents", but they look OK to me.

Please, rebase the change and I'll merge it.

Commit:
Add SiteName to Agent update payload

danielfurman commented:
> LGTM
> 
> I'm not sure how changes in cloud export examples (and related API code) are relevant to "synthetic agents", but they look OK to me.
> 
> Please, rebase the change and I'll merge it.

Changes in cloud exports are not related to agents, but I didn't bother creating separate PR for such small changes.
I rebased the branch and added SiteName field to update payload, because the API returns an error without it. It wasn't the case before.
